productcomments.php

```diff
@@ -666,6 +667,28 @@ public function getStandardFieldList()
         ];
     }
 
+    /**
+     * Renders author name for the list, with the link if the author is a customer.
+     *
+     * @param string $value
+     * @param array $row
+     *
+     * @return string
+     */
+    public function renderAuthorName($value, $row)
+    {
+        if (!empty($row['customer_id'])) {
+            $linkToCustomerProfile = $this->context->link->getAdminLink('AdminCustomers', false, [], [
+                'id_customer' => $row['customer_id'],
+                'viewcustomer' => 1,
+            ]);
+
+            return '<a href="' . $linkToCustomerProfile . '">' . $value . '</a>';
```

Comment:
Usually HTML escaping should be done here, not in the controller. But I understand changes should match legacy code.